### PR TITLE
Uniform bucket access

### LINKED ISSUE #34: Cannot deploy due to Uniform Bucket Access policy
I think the issue is this bucket: https://github.com/cloudspannerecosystem/autoscaler/blob/fbae879d5230e36baa4ed3beb12ef9df9e0541a5/terraform/modules/autoscaler/main.tf#L85

I have an organization policy which requires uniform bucket access. I cannot use this module without the `uniform_bucket_level_access` flag set `true`. Will submit a PR.

### PR DESCRIPTION
**Pull request by caddac**
resolves #34

### CODE REVIEW

**Comment by caddac** [comment]
@googlebot I signed it!

File: terraform/modules/autoscaler/main.tf [hunk 1]
```diff
@@ -27,14 +27,14 @@ resource "google_service_account" "scaler_sa" {
   display_name = "Autoscaler - Scaler Function Service Account"
 }
```
Comment (davidcueva) [inline terraform/modules/autoscaler/main.tf:29]: This change is unrelated to uniform bucket access.
Please revert it.

Comment (caddac) [reply terraform/modules/autoscaler/main.tf]: sorry about that, inadvertent change due to some testing I was doing.
